docs-partials/builder/common/AMIConfig-not-required.mdx

```diff
@@ -137,4 +137,7 @@
   If you specify a value for seconds, Amazon EC2 rounds the seconds to the nearest minute.
   You can’t specify a date in the past. The upper limit for DeprecateAt is 10 years from now.
 
+- `deregistration_protection` (DeregistrationProtectionOptions) - See [DeregistrationProtectionOptions](#deregistration-protection-options) below for more
```

Comment:
Since this doesn't unroll the contents of the generated docs, I would suggest manually adding them to the docs for each builder, ideally under a `# Deregistration Protection Options` section so the fragment can point to it.

Reply:
I just added to `docs/builders`. But cannot find a way to view on local machine. Hope it is correct.